part-spec/adc_dac.json

```diff
@@ -0,0 +1,167 @@
+                    "type": "array",
+                    "items": {
+                        "$ref": "definitions.json#/pinSpec"
+                    }
+                },
+                "package": {
+                    "description": "component's package size and description",
+                    "$ref": "definitions.json#/package"
+                }
+            }
+        },
+        {
+            "title": "dac",
+            "type": "object",
+            "required": [
+                "componentID"
+            ],
+            "properties": {
+                "componentID": {
+                    "description": "common component identifying information, such as mpn",
+                    "$ref": "definitions.json#/componentID"
+                },
+                "digitalResolution": {
+                    "description": "number of bits of resolution",
+                    "type": "number"
+                },
+                "offsetError": {
+                    "description": "analog resonse to an input code of all zeros",
+                    "comment": "units of percentage of full scale range",
+                    "$ref": "definitions.json#/unit"
+                },
+                "gainError": {
+                    "description": "difference (in percentage of FSR) of how well the slope of the actual transfer function matches the ideal transfer function",
+                    "comment": "units of percentage of full scale range",
+                    "$ref": "definitions.json#/unit"
+                },
+                "integralNonlinearity": {
+                    "description": "deviation of an actual transfer function from an ideal transfer function, in LSB",
+                    "comment": "units of LSB",
+                    "type": "number"
+                },
+                "differentialNonlinearity": {
+                    "description": "difference between the ideal and the actual output responses for successive DAC codes, in LSB",
+                    "comment": "units of +/- LSB",
+                    "type": "number"
+                },
+                "settlingTime": {
+                    "description": "time from application of input code to valid output response",
+                    "comment": "units of seconds",
+                    "$ref": "definitions.json#/unit"
+                },
+                "interface": {
+                    "description": "digital communication interface",
+                    "examples": [
+                        "spi",
+                        "i2c"
+                    ],
+                    "type": "string"
+                },
+                "otuputType": {
+                    "description": "whether the DAC has a single ended or differential output",
+                    "enum": [
+                        "singleEnded",
+                        "differential"
+                    ],
+                    "type": "string"
+                },
```

Comment:
properties to add: output voltage range, sampling rate

Reply:
I think output voltage rate is covered by pins, but I added sampling rate.